Bixal/bixal-site-drupal
# Feature: Create blog stories

> [!IMPORTANT]  
> Major re-org in stories (sorry). This reorg helps us cleanly create pages and reuse components.

Create pages for Blog Landing and Blog Post pages. Refinements to Hero and Blurbs. Closes #18. Closes #21.


## Major changes

- Created local storybook command via `storybook:local` - 4b5d910.
- Moved Components and Pages to `Stories` directory. Fixed aliases - 53bde57b35f7cc819cfcef61f3bf6d46629426b3.
- Refined Hero styles.
- Major refinements to Blurb and Blurb Collections - cb4e9d137c23a4079e5eca538f1b86f254448989.
- Created Blog Landing page - 0eaaf1eb5e791d8aac261df297958dd760b316d4.
- Created Blog post page - d7c153d9082a933b3d8882dff952d7d812ae4201.

## Additional information

You can now use `@components` and `@pages` aliases in twig templates.

**Example**

```twig
  
  {% include "@components/blurb/blurb.html.twig" with blurb %}

  
  {% extends "@pages/base/base.html.twig" %}
```


## How to test

1. Run StorybookJS
2. Visit Blog Landing page - `http://localhost:5050/?path=/story/pages-blog-landing--default`.
3. Confirm elements and general styling.
4. Visit Blog post page - `http://localhost:5050/?path=/story/pages-blog-post--default`.
5. Confirm elements and general styling.
